boards/kaga-fei/ec4l15ba1/board.cmake

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# Copyright (c) 2025 KAGA FEI Co., Ltd.
+# SPDX-License-Identifier: Apache-2.0
+
+if(CONFIG_SOC_NRF54L05_CPUAPP OR CONFIG_SOC_NRF54L10_CPUAPP OR CONFIG_SOC_NRF54L15_CPUAPP)
+  board_runner_args(jlink "--device=cortex-m33" "--speed=4000")
+elseif(CONFIG_SOC_NRF54L05_CPUFLPR OR CONFIG_SOC_NRF54L10_CPUFLPR OR CONFIG_SOC_NRF54L15_CPUFLPR)
+  set(JLINKSCRIPTFILE ${CMAKE_CURRENT_LIST_DIR}/support/nrf54l_05_10_15_cpuflpr.JLinkScript)
+  board_runner_args(jlink "--device=RISC-V" "--speed=4000" "-if SW" "--tool-opt=-jlinkscriptfile ${JLINKSCRIPTFILE}")
```

Comment:
this is outdated, please look at https://github.com/zephyrproject-rtos/zephyr/tree/main/boards/nordic/nrf54l15dk for changes and update the whole PR accordingly https://github.com/zephyrproject-rtos/zephyr/tree/main/boards/nordic/nrf54l15dk

Note: this module has an nrf54l15, no reference or usage of nrf54l05 or nrf54l10 to be in this PR